modules/ROOT/pages/kubernetes/accessing-neo4j.adoc

```diff
@@ -344,6 +349,257 @@ For a detailed example, see xref:kubernetes/quickstart-cluster/access-outside-k8
+
+image::reverse-proxy.svg[title="Reverse proxy flow diagram"]
+
+The reverse proxy creates an HTTP server, which routes requests to either the Bolt reverse proxy or HTTP reverse proxy based on the request headers.
+Upon receiving a response, the Bolt reverse proxy updates the response to replace the Bolt port with either `:80` or `:443`.
+
+==== Configuration options
+
+To see all configurable options, run the following command:
+
+[source, shell]
+--
+helm show values neo4j/neo4j-reverse-proxy
+--
+
+[source, yaml]
+----
+# Default values for neo4j reverse proxy helm chart
+
+## @param nameOverride String to partially override common.names.fullname
+nameOverride: ""
+## @param fullnameOverride String to fully override common.names.fullname
+fullnameOverride: ""
+
+# Parameters for reverse proxy
+reverseProxy:
+  image: "neo4j/neo4j-reverse-proxy"
+
+  # Name of the kubernetes service. This service should have the ports 7474 and 7687 open.
+  # This could be the admin service ex: "standalone-admin" or the loadbalancer service ex: "standalone" created via the neo4j helm chart
+  # serviceName , namespace , domain together will form the complete k8s service url. Ex: standalone-admin.default.svc.cluster.local
+  # When used against a cluster ensure the service being used is pointing to all the cluster instances.
+  # This could be the loadbalancer from neo4j helm chart or the headless service installed via neo4j-headless-service helm chart
+  serviceName: ""
+  # default is set to cluster.local
+  domain: "cluster.local"
+
+  # This assumes ingress-nginx controller or haproxy-ingress-controller is already installed in your kubernetes cluster.
+  # You can install ingress-nginx by following instructions on this link https://github.com/kubernetes/ingress-nginx/blob/main/docs/deploy/index.md#quick-start
+  # You can install haproxy-ingress by following instructions on this link https://haproxy-ingress.github.io/docs/getting-started/
+  ingress:
+    enabled: true
+    #default value is nginx. It can be either nginx or haproxy
+    className: nginx
+    annotations: {}
+#      "demo": "value"
+#      "demo2": "value2"
+    tls:
+      enabled: false
+      config: []
+#      - secretName: "demo2"
+#        hosts:
+#          - localhost
+----
+
+==== Install the Helm chart
+
+The following steps assume that you have a Kubernetes cluster running and a standalone Neo4j Helm chart installed.
+The standalone Neo4j has a Neo4j service with the name _standalone-admin_, and it has `:7474` an `:7687` opened.
+To verify that, run:
+[source, shell]
+----
+kubectl get all, pvc, pv, configmaps, secrets
+----
+
+You also need to have an Ingress controller for the Kubernetes Ingress to work.
+The following steps use the Nginx Ingress controller.
```

Comment:
we should make Nginx Ingress Controller bold to highlight it

Reply:
Hmm, why do we need to highlight it? We use italics when we want to emphasize something and bold only for crucial information, which, if missed, might lead to some unfortunate consequences.

Reply:
Italics did not got my attention and its important to highlight that we have used nginx ingress controller for this example